### PR TITLE
Possibility to send value to a transaction

### PR DESCRIPTION
**Pull request by antho1404**
It is now possible when executing a task to attach a value and send ETH

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,12 +16,12 @@ const eventsHandler = require('./events')({
 
 const contract = new web3.eth.Contract(ABI, process.env.CONTRACT_ADDRESS)
 
-const execute = async ({ method, privateKey, inputs }) => {
+const execute = async ({ method, privateKey, inputs, value }) => {
   const account = web3.eth.accounts.privateKeyToAccount(privateKey)
```
Comment (NicolasMahe) [inline index.js]: I really recommend to add a check on the content of value.
BN.js is ONLY compatible with Integer. And I think it just truncate the decimals if there is any without throwing error.

Comment (NicolasMahe) [reply index.js]: And maybe it's also only compatible with string. I'm not sure.

Comment (antho1404) [reply index.js]: updated to bignumber

File: mesg.yml [hunk 2]
```diff
@@ -18,6 +18,9 @@ tasks:
       inputs:
         repeated: true
         type: Any
+      value:
+        type: Any
```
Comment (antho1404) [inline mesg.yml]:
```suggestion
        type: String
```

**Pull request merged**